{"uuid":"71fdcc41-15a5-4759-952d-0a3a2af1db6f","historyId":"8bf9408689cde93e0668e657e595ffe7","fullName":"Array: The user is presented with error message for code with invalid syntax in tryEditor for Find Numbers with Even Number of Digits","labels":[{"name":"tag","value":"arrays"},{"name":"host","value":"LAPTOP-RSES787T"},{"name":"thread","value":"21960@LAPTOP-RSES787T.main(1)"},{"name":"feature","value":"Array"},{"name":"story","value":"The user is presented with error message for code with invalid syntax in tryEditor for Find Numbers with Even Number of Digits"},{"name":"suite","value":"Array"},{"name":"testClass","value":"The user is presented with error message for code with invalid syntax in tryEditor for Find Numbers with Even Number of Digits"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/5arrays.feature"},{"name":"package","value":"src.test.resources.features.5arrays_feature.Array"}],"links":[],"name":"The user is presented with error message for code with invalid syntax in tryEditor for Find Numbers with Even Number of Digits","status":"passed","stage":"finished","steps":[{"name":"Given  The user is on \"Question\" page of \"Find Numbers with Even Number of Digits\" after logged in","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673382249954,"stop":1673382250276},{"name":"When  The user enter python code with invalid syntax in tryEditor from sheet \"pythonCode\" and 1","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673382250276,"stop":1673382250973},{"name":"And  The user clicks on run button","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673382250986,"stop":1673382251270},{"name":"Then  The user should be presented with error message as \"SyntaxError: bad input on line 1\"","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673382251271,"stop":1673382251287}],"attachments":[],"parameters":[{"name":"Sheetname","value":"pythonCode"},{"name":"RowNumber","value":"1"}],"start":1673382249953,"stop":1673382251288}